From johnpol@2ka.mipt.ru Thu Apr 27 08:40:59 2006
Date: Thu, 27 Apr 2006 19:40:27 +0400
From: Evgeniy Polyakov <johnpol@2ka.mipt.ru>
To: GregKH <greg@kroah.com>
Subject: w1: clean up W1_CON dependency.
Message-ID: <20060427154026.GB31564@2ka.mipt.ru>
Content-Disposition: inline

If w1 is not enabled, w1_con should not appear in configuration,
even if no logic is turned on without w1.
W1_CON should depend on W1 also.

Signed-off-by: Randy Dunlap <rdunlap@xenotime.net>
Signed-off-by: Evgeniy Polyakov <johnpol@2ka.mipt.ru>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/w1/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/w1/Kconfig
+++ gregkh-2.6/drivers/w1/Kconfig
@@ -12,7 +12,7 @@ config W1
 	  will be called wire.ko.
 
 config W1_CON
-	depends on CONNECTOR
+	depends on CONNECTOR && W1
 	bool "Userspace communication over connector"
 	default y
 	--- help ---
